src/vm/dllimport.cpp

```diff
@@ -2950,6 +2960,236 @@ void PInvokeStaticSigInfo::DllImportInit(MethodDesc* pMD, LPCUTF8 *ppLibName, LP
     }
 }
 
+#if !defined(CROSSGEN_COMPILE) // IJW
+
+// This attempts to guess whether a target is an API call that uses SetLastError to communicate errors.
+static BOOL HeuristicDoesThisLooksLikeAnApiCallHelper(LPBYTE pTarget)
+{
+    CONTRACTL
+    {
+        NOTHROW;
+        GC_NOTRIGGER;
+        MODE_ANY;
+    }
+    CONTRACTL_END;
+
+    // This code is not that useful anymore since this functionality is already embedded in the VC linker.
```

Comment:
I am wondering whether we should delete this code from CoreCLR completely. It looks like a very old compat quirk.

Reply:
Looks like we probably can. I've confirmed the C++ compiler is putting the lasterror flag in. Since these were causing Unix build failures anyway, I've removed them.